pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -46,6 +46,16 @@ import (
 // Name of the plugin used in the plugin registry and configurations.
 const Name = names.DefaultPreemption
 
+// IsEligiblePodFunc is a function which may be assigned to the DefaultPreemption plugin.
+// This function returns whether a given victim pod on the provided nodeInfo is eligible
+// to be preempted by the provided preemptor.
+type IsEligiblePodFunc func(nodeInfo *framework.NodeInfo, victim *framework.PodInfo, preemptor *v1.Pod) bool
+
+// MoreImportantPodFunc is a function which may be assigned to the DefaultPreemption plugin.
+// This function returns true if the priority of the first pod is higher than the second pod.
```

Comment:
```suggestion
// The implementations should return true if the first pod is more important than the second pod and the second one should be considered for preemption before the first one.
```

Overwritten behavior does not need to look into priorities